rmw_fastrtps_dynamic_cpp/src/rmw_serialize.cpp

```diff
@@ -48,7 +48,8 @@ rmw_serialize(
     }
   }
 
-  eprosima::fastcdr::FastBuffer buffer(reinterpret_cast<char *>(serialized_message->buffer), data_length);
+  eprosima::fastcdr::FastBuffer buffer(reinterpret_cast<char *>(serialized_message->buffer),
+    data_length);
```

Comment:
Was this linter error preexisting?

nitpick: we usually wrap all or none (no leaving the first parameter on the same line as the call)

Reply:
It was existing when I got to it, will rewrap accordingly.